INFO - __main__ (10:42:04.078) - MainThread (3572):
Starting NVDA version 2019.3.1
INFO - core.main (10:42:05.375) - MainThread (3572):
Config dir: C:\Users\User\AppData\Roaming\nvda
INFO - config.ConfigManager._loadConfig (10:42:05.375) - MainThread (3572):
Loading config: C:\Users\User\AppData\Roaming\nvda\nvda.ini
INFO - core.main (10:42:05.587) - MainThread (3572):
Using Windows version 10.0.17763 workstation
INFO - core.main (10:42:05.587) - MainThread (3572):
Using Python version 3.7.5 (tags/v3.7.5:5c02a39a0b, Oct 14 2019, 23:09:19) [MSC v.1916 32 bit (Intel)]
INFO - core.main (10:42:05.587) - MainThread (3572):
Using comtypes version 1.1.7
INFO - core.main (10:42:05.587) - MainThread (3572):
Using configobj version 5.1.0 with validate version 1.0.1
INFO - synthDrivers.espeak.SynthDriver.__init__ (10:42:06.951) - MainThread (3572):
Using eSpeak NG version b'1.49.3 dev'
INFO - synthDriverHandler.setSynth (10:42:08.037) - MainThread (3572):
Loaded synthDriver espeak
INFO - core.main (10:42:08.423) - MainThread (3572):
Using wx version 4.0.3 msw (phoenix) wxWidgets 3.0.5 with six version 1.12.0
INFO - brailleInput.initialize (10:42:08.423) - MainThread (3572):
Braille input initialized
INFO - braille.initialize (10:42:08.423) - MainThread (3572):
Using liblouis version 3.10.0
INFO - braille.initialize (10:42:08.442) - MainThread (3572):
Using pySerial version 3.4
INFO - braille.BrailleHandler.setDisplayByName (10:42:08.451) - MainThread (3572):
Loaded braille display driver noBraille, current display has 0 cells.
INFO - brailleDisplayDrivers.freedomScientific.BrailleDisplayDriver.__init__ (10:42:08.469) - braille._BgThread (316):
Found Focus 40 connected via custom (\\?\usb#vid_0f4e&pid_0114#1234567#{a5dcbf10-6530-11d2-901f-00c04fb951ed})
INFO - braille.BrailleHandler.setDisplayByName (10:42:08.480) - braille._BgThread (316):
Loaded braille display driver freedomScientific, current display has 40 cells.
INFO - core.main (10:42:09.040) - MainThread (3572):
Java Access Bridge support initialized
INFO - _UIAHandler.UIAHandler.MTAThreadFunc (10:42:09.079) - _UIAHandler.UIAHandler.MTAThread (8004):
UIAutomation: IUIAutomation6
ERROR - globalPluginHandler.listPlugins (10:42:09.940) - MainThread (3572):
Error importing global plugin ocr
Traceback (most recent call last):
  File "globalPluginHandler.pyc", line 23, in listPlugins
  File "importlib\__init__.pyc", line 127, in import_module
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Users\User\AppData\Roaming\nvda\addons\ocr\globalPlugins\ocr\__init__.py", line 37, in <module>
    from .PIL import ImageGrab
  File "C:\Users\User\AppData\Roaming\nvda\addons\ocr\globalPlugins\ocr\PIL\ImageGrab.py", line 23, in <module>
    from . import Image
  File "C:\Users\User\AppData\Roaming\nvda\addons\ocr\globalPlugins\ocr\PIL\Image.py", line 39, in <module>
    from pathlib import Path
ModuleNotFoundError: No module named 'pathlib'
INFO - core.main (10:42:10.424) - MainThread (3572):
NVDA initialized
ERROR - scriptHandler.executeScript (10:49:57.021) - MainThread (3572):
error executing script: <bound method EditableText.script_caret_newLine of <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionBrokenFocusedStateEditorMozillaIAccessible object at 0x00FDBC10>> with gesture 'intro'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 205, in executeScript
  File "editableText.pyc", line 188, in script_caret_newLine
  File "editableText.pyc", line 94, in _hasCaretMoved
  File "documentBase.pyc", line 24, in makeTextInfo
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 94, in __init__
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 202, in _findContentDescendant
  File "comtypesMonkeyPatches.pyc", line 26, in __call__
_ctypes.COMError: (-2147467262, 'Interfaz no compatible', (None, None, None, 0, None))
ERROR - scriptHandler.executeScript (10:57:36.468) - MainThread (3572):
error executing script: <bound method EditableText.script_caret_newLine of <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionBrokenFocusedStateEditorMozillaIAccessible object at 0x053B50F0>> with gesture 'intro'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 205, in executeScript
  File "editableText.pyc", line 188, in script_caret_newLine
  File "editableText.pyc", line 94, in _hasCaretMoved
  File "documentBase.pyc", line 24, in makeTextInfo
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 94, in __init__
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 202, in _findContentDescendant
  File "comtypesMonkeyPatches.pyc", line 26, in __call__
_ctypes.COMError: (-2147467262, 'Interfaz no compatible', (None, None, None, 0, None))
INFO - core.main (11:00:32.313) - MainThread (3572):
Exiting
INFO - config.ConfigManager.save (11:00:32.356) - MainThread (3572):
Base configuration saved
INFO - __main__ (11:00:33.664) - MainThread (3572):
NVDA exit
